library/src/main/java/com/chuckerteam/chucker/internal/ui/error/ErrorActivity.kt

```diff
@@ -15,7 +15,6 @@ import com.chuckerteam.chucker.internal.data.entity.RecordedThrowable
 import com.chuckerteam.chucker.internal.data.repository.RepositoryProvider
 import java.text.DateFormat
 
-private const val EXTRA_THROWABLE_ID = "EXTRA_THROWABLE_ID"
 private const val TEXT_PLAIN = "text/plain"
```

Comment:
Sure. I just thought that classes had such constants declaration, because you prefer to avoid companion objects when possible, like some devs do (for example: https://medium.com/@BladeCoder/exploring-kotlins-hidden-costs-part-1-fbb9935d9b62 ).
As to this particular constant for `TEXT_PLAIN` - I wanted to remove it in another PR, but for now will just move it.

Reply:
Thanks for the link 👍